nextstrain/cli/command/view.py

```diff
@@ -37,14 +37,22 @@ def register_parser(subparser):
 
     parser.add_argument(
         "--allow-remote-access",
-        help   = "Allow other computers on the network to access the website",
-        action = "store_true")
+        help   = "Allow other computers on the network to access the website (alias for --host=0.0.0.0)" + SKIP_AUTO_DEFAULT_IN_HELP,
+        dest   = "host",
+        action = "store_const",
+        const  = "0.0.0.0",
+        default = SUPPRESS)
```

Comment:
Somehow `nextstrain view --help` is failing with this. Replacing `SUPPRESS` with the literal `'==SUPPRESS=='` fixes it, not sure why 🤔

Reply:
What's the error output?

Reply:
```
$ nextstrain view --help
Traceback (most recent call last):
  File "/home/ubuntu/miniconda3/bin/nextstrain", line 33, in <module>
    sys.exit(load_entry_point('nextstrain-cli', 'console_scripts', 'nextstrain')())
  File "/home/ubuntu/cli/nextstrain/cli/__main__.py", line 10, in main
    return cli.run( argv[1:] )
  File "/home/ubuntu/cli/nextstrain/cli/__init__.py", line 28, in run
    opts = parser.parse_args(args)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 1818, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 1851, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 2042, in _parse_known_args
    positionals_end_index = consume_positionals(start_index)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 2019, in consume_positionals
    take_action(action, args)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 1928, in take_action
    action(self, namespace, argument_values, option_string)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 1207, in __call__
    subnamespace, arg_strings = parser.parse_known_args(arg_strings, None)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 1851, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 2060, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 2000, in consume_optional
    take_action(action, args, option_string)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 1928, in take_action
    action(self, namespace, argument_values, option_string)
  File "/home/ubuntu/cli/nextstrain/cli/argparse.py", line 100, in __call__
    full_help  = parser.format_help()
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 2532, in format_help
    return formatter.format_help()
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 283, in format_help
    help = self._root_section.format_help()
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 214, in format_help
    item_help = join([func(*args) for func, args in self.items])
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 214, in <listcomp>
    item_help = join([func(*args) for func, args in self.items])
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 214, in format_help
    item_help = join([func(*args) for func, args in self.items])
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 214, in <listcomp>
    item_help = join([func(*args) for func, args in self.items])
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 530, in _format_action
    help_text = self._expand_help(action)
  File "/home/ubuntu/miniconda3/lib/python3.9/argparse.py", line 626, in _expand_help
    return self._get_help_string(action) % params
KeyError: 'default'
```

Confirmed `'default'` being the value of `SKIP_AUTO_DEFAULT_IN_HELP`

Reply:
Oh, whoops. I used `SKIP_AUTO_DEFAULT_IN_HELP` first to avoid the default `(default: None)` that was appearing in help, and then added `default = SUPPRESS` later to avoid setting `opts.host` to `None` entirely.

Reply:
Fixed.

Reply:
To avoid issues like this in the past, I've [written tests that run every command's `--help`](https://github.com/seattleflu/id3c/blob/6b3bea04031495ef2b971daa2282b759104d15b5/tests/help.py) to make sure it doesn't crash. Might be worth doing something similar here. :-)